e2e-tests/commands/insurance/account/assert-password-reveal-button.js

```diff
@@ -8,13 +8,16 @@ const {
 
 const FIELD_STRINGS = ACCOUNT_FIELDS[FIELD_ID];
 
-const expectedShowText = `${FIELD_STRINGS.REVEAL.SHOW} password`;
-const expectedHideText = `${FIELD_STRINGS.REVEAL.HIDE} password`;
+const EXPECTED_ARIA_LABELS = {
```

Comment:
not a blocker, should these be constants instead?

Reply:
Good shout, although currently, it's only relevant to this E2E test - this text is automatically generated by the password component.

I think we should keep this in here for now. I can't see a scenario where this specific password reveal copy would be used somewhere else.